client/my-sites/media-library/test/data-source.jsx

```diff
@@ -36,14 +36,15 @@ describe( 'MediaLibraryDataSource', () => {
 					/>
 				</ReduxProvider>
 			);
-			expect( wrapper.find( 'button[data-source="google_photos"]' ) ).toHaveLength( 1 );
-			expect( wrapper.find( 'button[data-source="pexels"]' ) ).toHaveLength( 1 );
+
+			expect( screen.getByText( 'Google Photos' ) ).toBeInTheDocument();
```

Comment:
What about using `queryBy` instead of `getBy` methods when asserting 
  with `.toBeInTheDocument` because `getBy` will throw an error of itself if the element isn't found.  `expect( screen.queryByText( 'WordPress.com' ) ).toBeInTheDocument(); instead of expect( screen.getByText( 'WordPress.com' ) ).toBeInTheDocument()`

Reply:
![](https://d.pr/i/ifcP95+)

Reply:
updated it